packages/core/src/shared/languageServer/types.ts

```diff
@@ -15,8 +15,16 @@ export interface LspResult {
     assetDirectory: string
 }
 
+export interface ExecutablePaths {
```

Comment:
nit: Will these paths always be executables? Would something like `ResourcePaths` be more generalized for the long term?

Reply:
Yeah that's a way better name, I couldn't think of a good one 😆